File: 9/main.java
class Solution {
    public static int timeTravel(int n, int[] arr) {
        int ans = 0;
        for(int i = 1; i < n; i++)
            ans += (arr[i] > arr[i - 1] ? 1 : (arr[i] < arr[i - 1] ? 2 : 0));
        return ans;
    }
}

File: 9/main.py
#Back-end complete function Template for Python 3
class Solution:
    def timeTravel(self, n, arr):
        ans=0
        for i in range(1,n):
            if (arr[i]>arr[i-1]):
                ans+=1
            elif (arr[i]<arr[i-1]):
                ans+=2
        return ans
